### PR TITLE
NYC WDI 52 installfest config

### PR DESCRIPTION
**Pull request by jasonseminara**
We've removed the browser installation since we realized we don't use firefox regularly, and for the minority of students who don't have Chrome installed, we can run `brew cask install chrome` individually.

### CODE REVIEW

**Comment by h4w5** [comment]
Hi Jason!

These are all very good points. Two issues arise, though:

1. there can be only one *canonical* script, and everyone needs to be on board for it, and
2. therefore I try to make the script ***maximal***.

I am kind of loathe to remove things, in general, because everything in there has been requested to be in there by someone. For example: **mac2imgur** is really important to how we run our class in LA, and it's definitely staying in!

However, I think it makes sense to drop the browser installs, for now, for just the issue you brought up. I am in the process of trying to create a sub-script to check for the presence of some apps, that will install if they aren't already installed, but everything on my plate has been bogged down.

FYI, for us the big double install issue comes from Sublime Text. I won't drop that one, however, since the script also does some needed setup for Sublime.

Next: I have a question about:

```
# OSX replacements
bash
```

Does Homebrew automatically link this? I know if symlinks binaries generally, but does it also correctly symlink the shell?

Also, for all of those tools, what is the reason for the upgrade? Are you using any new functionality? Is it for security, best practices, etc.?

Everything else looks good, except for setting the default editor for git to Sublime. I've found it MUCH easier to teach the class to get used to dealing with git messages in the terminal. git taking over Sublime is just a headache.

If you want to switch to using it, then you can definitely do that *after Installfest*, or you can create a new branch with that change in it. I really want to create a version of this that allows us to generate and save multiple versions of the script, but it's not there yet and I'm not getting much support to implement that.

In summary:

I accept all the changes except:

- the removal of  mac2imgur, jumpcut and xquartz
- changing the git editor to Sublime from nano

Sound good?

**Comment by h4w5** [comment]
@jasonseminara this integrates some of the thoughts above (PR #116). I am worried about changing bash (and other core utils) especially since if there is another ShellShock-style vulnerability, and Mac pushes updates to the users, they may not realize that they are using separate versions that need updating... Is there is good reason to update these tools that overrides that fear?

**Comment by jasonseminara** [comment]
>git taking over Sublime is just a headache.

agreed. I'll revert that. 
As for the GNU tools, I have no scientific evidence except for the promise of newer shinier tools (I haven't noticed any differences).  

AFAIK, brew does a good job of symlinking bash 4, and once you restart your bash sesh you'll be in bash-4.0
